doc/attributes.xml

```diff
@@ -815,52 +816,66 @@ gap> RZMSConnectedComponents(R);
   <Attr Name="RZMSNormalization" Arg="R"/>
```

Comment:
Argh, I see that it is probably in the line after the last line of this diff, good job @wilfwilson